outputs.tf

```diff
@@ -27,3 +27,8 @@ output "subnet" {
   value       = local.subnet
   description = "The IP space for the cluster"
 }
+
+output "cluster_name" {
+  value = local.cluster_name
```

Comment:
fix the formatting... you can run `terraform fmt --recursive` to do this for you.

Reply:
Done!